lib/src/Base/Algo/GaussKronrod.cxx

```diff
@@ -88,6 +88,7 @@ Point GaussKronrod::integrate(const Function & function,
                               Point & ei) const
 {
   if (function.getInputDimension() != 1) throw InvalidArgumentException(HERE) << "Error: can integrate only 1D function, here input dimension=" << function.getInputDimension();
+  if (!(b >= a)) throw InvalidArgumentException(HERE) << "GaussKronrod: a (" << a << ") should be lower than b (" << b << ")";
```

Comment:
Why do you want to enforce this? There is no problem in evaluating an integral with a>=b, and it is even needed for backward integration!

Reply:
I was fixing a distribution bug, and thought that GaussKronrod assumed a<b:
```
const Scalar width = 0.5 * (b - a);
  const Scalar center = 0.5 * (a + b);
```